hr_timesheet_no_closed_project_task/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License AGPL-3
+
+===================================
+HR Timesheet No Closed Project-Task
+===================================
+
+This module prevents to select closed project or task on timesheet line.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Click on install button
+
+Usage
+=====
+
```

Comment:
Add info about usage as you told me in https://github.com/OCA/hr-timesheet/pull/70#issuecomment-230165731